### PR TITLE
Fix frida_gdiplus

### PR DESCRIPTION
**Pull request by tokatoka**
Basically I got status_wait_0 but it's not handled.

Create a default handler routine
and if the exception code is not in crash list, then fall back to the default handler

### CODE REVIEW

**Comment by tokatoka** [comment]
@expend20 
Hi expy.
If you can help, can you check I'm doing correct here?

Basically I think we want a default exception handler for first-chance exception(?) here right?

File: libafl/src/executors/inprocess.rs [hunk 1]
```diff
@@ -1007,6 +1002,7 @@ mod windows_exception_handler {
         }
 
         fn exceptions(&self) -> Vec<ExceptionCode> {
+            assert!(CRASH_EXCEPTIONS.to_vec().len() < EXCEPTION_HANDLERS_SIZE);
```
Comment (domenukk) [inline libafl/src/executors/inprocess.rs:1005]: Doing to_vec() twice makes no sense, right?
We can probably even just use (and return) a `'const` slice here

File: libafl/src/executors/inprocess.rs [hunk 2]
```diff
@@ -1172,7 +1170,14 @@ mod windows_exception_handler {
                     .0,
             )
             .unwrap();
-            eprintln!("Crashed with {code}");
+
+            let exception_list = data.exceptions();
+            if exception_list.contains(&code) {
+                eprintln!("Crashed with {code}");
+            } else {
+                eprintln!("Exception code received, but {code} is not in CRASH_EXCEPTIONS");
```
Comment (domenukk) [inline libafl/src/executors/inprocess.rs:1178]: Would it be good to change the crashing signals at runtime? So, V8 needs to handle sigtrap on linux

File: libafl/src/executors/inprocess.rs (hunk 1 above)
Comment (tokatoka) [reply libafl/src/executors/inprocess.rs]: fixed

Comment (domenukk) [reply libafl/src/executors/inprocess.rs]: Still returns a Vec needlessly but fine, shouldn't be on the hot path, so copying a few pointers who cares in this case

**Comment by expend20** [comment]
Hello, 
I'm not really sure you really need EXCEPTION_EXECUTE_HANDLER, what happens if there is no handler in the original code? Maybe EXCEPTION_CONTINUE_SEARCH is better option, but can't really confirm this. Otherwise looks good to me

**Comment by tokatoka** [comment]
Yes true

**Pull request merged**